eureka-client/src/main/java/com/netflix/discovery/DiscoveryClient.java

```diff
@@ -1349,10 +1349,8 @@ public String getId() {
 
                 @Override
                 public void notify(StatusChangeEvent statusChangeEvent) {
-                    if (InstanceStatus.DOWN == statusChangeEvent.getStatus() ||
-                            InstanceStatus.DOWN == statusChangeEvent.getPreviousStatus()) {
-                        // log at warn level if DOWN was involved
-                        logger.warn("Saw local status change event {}", statusChangeEvent);
+                    if (InstanceStatus.DOWN == statusChangeEvent.getStatus()) {
```

Comment:
Do you mind also changing the this Yoda condition?

Reply:
Done